minisom.py

```diff
@@ -307,7 +314,7 @@ def winner(self, x):
         return unravel_index(self._activation_map.argmin(),
                              self._activation_map.shape)
 
-    def update(self, x, win, t, max_iteration):
+    def update(self, x, win, decay_rate, max_iteration):
```

Comment:
This is a public interface and changing the name will not make the new version retrocompatible. Let's keep it as `t` and have an updated documentation.